migrations/v2.js

```diff
@@ -0,0 +1,41 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+const getCourse = content => {
+  const course = content.find(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._menu?._boxMenu;
+};
+
+describe('Box menu - v2.0.2 to v2.0.3', async () => {
+
+  // https://github.com/adaptlearning/adapt-contrib-boxMenu/compare/v2.0.2..v2.0.3
+
+  let course, courseBoxMenuGlobals;
+  const durationLabel = 'Duration:';
+
+  whereFromPlugin('Box menu - from v2.0.2', { name: 'adapt-contrib-boxMenu', version: '<2.0.3' });
```

Comment:
Just so we're not migrating anything earlier than v2
```suggestion
  whereFromPlugin('Box menu - from v2.0.2', { name: 'adapt-contrib-boxMenu', version: '>=2.0.0 <2.0.3' });
```